lib/main.dart

```diff
@@ -17,10 +21,17 @@ class LachiesLifePlannerApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      debugShowCheckedModeBanner: false,
-      title: 'Lachie\'s Life Planner',
-      home: HomeScreen(),
+    return MultiBlocProvider(
+      providers: [
+        BlocProvider<TasksBloc>(
+          create: (context) => TasksBloc(tasksRepository: TasksRepository())..add(LoadTasksEvent()),
+        )
```

Comment:
Is there a reason that we are loading tasks upon opening the app?  
Should that be done on the TasksScreen?
Why are we adding an event straight away?

Reply:
What this does is:
- Adds an initial event for the `TaskBloc`
- This event is actually dispatched upon the `TaskBloc` widget being loaded, hence on the `TasksScreen`
- If we do not add this event here, then we would have to add it somewhere else (probably also on the `TasksScreen`)
- But performance is probably best if it is here.

In summary: I will leave this as it is